javascript/react/src/components/verbs/forms/OrderbyForm/OrderbyForm.tsx

```diff
@@ -21,6 +21,25 @@ import { Container, icons, Sorts } from './OrderbyForm.styles.js'
 export const OrderbyForm: React.FC<StepFormProps<OrderbyArgs>> = memo(
 	function OrderbyForm({ step, workflow, input, onChange }) {
 		const dataTable = useStepInputTable(step, workflow, input)
+
+		useEffect(() => {
+			if (dataTable != null && onChange != null) {
+				if (!step.args.orders || step.args.orders.length === 0) {
+					onChange({
+						...step,
+						args: {
+							orders: [
+								{
+									column: dataTable.columnNames()[0] || '',
```

Comment:
Is catching a zero-column case here important?

Reply:
Sort of - the issue is that `column` is required for the verb per the schema, but a partial should be fine during UX building. We've always had a little bit of disconnect in various places like this where validation should happen, say, before allowing the user to save a step, but in the meantime _all_ properties on verb args could be Partial<T>